test/test_mesh.py

```diff
@@ -36,3 +39,81 @@ def test_vdim_changes_when_mesh_changes():
     for mesh in [mesh_1D, mesh_2D, mesh_3D]:
         my_mesh.mesh = mesh
         assert my_mesh.vdim == mesh.topology.dim
+
+
+@pytest.mark.parametrize("mesh", [mesh_1D, mesh_2D, mesh_3D])
+def test_meshtags_from_xdmf(tmp_path, mesh):
+    """Test that the facet and volume meshtags are read correctly from the mesh XDMF files"""
+    # create mesh functions
+    fdim = mesh.topology.dim - 1
+    vdim = mesh.topology.dim
+
+    # create facet meshtags
+    facet_indices = []
+    for i in range(1):
+        facet_indices.append(
+            fenics_mesh.locate_entities_boundary(
+                mesh, fdim, lambda x: np.isclose(x[i], 0)
+            )
+        )
+        facet_indices.append(
+            fenics_mesh.locate_entities_boundary(
+                mesh, fdim, lambda x: np.isclose(x[i], 1)
+            )
+        )
```

Comment:
can we improve the readibility a bit?
maybe by creating intermediate variables?